zigzag/zigzag_test_log.py

```diff
@@ -529,6 +542,17 @@ def _lookup_ids(self):
         except IndexError:  # test case has not been created yet in qTest
             pass
 
+    def _lookup_job_config_attributes(self):
+        """ Finds the array of job config attributes on the end of the job name.
+
+        Returns:
+            List: of job config attributes.
+        """
+        full_name = self._testcase_xml.items()[3][1]
+        under_delimited_list = full_name[full_name.find("[")+1:full_name.find("]")]
```

Comment:
Are we sure that the list is under_delimited?  In Tempest the values are comma delimited `name="test_list_user_projects[id-a831e70c-e35b-430b-92ed-81ebbc5437b8]"`